internal/peer/node/rebuild_dbs_test.go

```diff
@@ -26,33 +20,9 @@ func TestRebuildDBsCmd(t *testing.T) {
 	viper.Set("peer.fileSystemPath", testPath)
 	defer os.RemoveAll(testPath)
 
-	viper.Set("logging.ledger", "INFO")
-	rootFSPath := filepath.Join(config.GetPath("peer.fileSystemPath"), "ledgersData")
-	bookkeeperDBPath := kvledger.BookkeeperDBPath(rootFSPath)
-	configHistoryDBPath := kvledger.ConfigHistoryDBPath(rootFSPath)
-	historyDBPath := kvledger.HistoryDBPath(rootFSPath)
-	stateDBPath := kvledger.StateDBPath(rootFSPath)
-	blockstoreIndexDBPath := filepath.Join(kvledger.BlockStorePath(rootFSPath), "index")
-
-	dbPaths := []string{bookkeeperDBPath, configHistoryDBPath, historyDBPath, stateDBPath, blockstoreIndexDBPath}
-	for _, dbPath := range dbPaths {
-		require.NoError(t, os.MkdirAll(dbPath, 0755))
-		require.NoError(t, ioutil.WriteFile(path.Join(dbPath, "dummyfile.txt"), []byte("this is a dummy file for test"), 0644))
-	}
-
-	// check dbs exist before upgrade
-	for _, dbPath := range dbPaths {
-		_, err := os.Stat(dbPath)
-		require.False(t, os.IsNotExist(err))
-	}
-
+	// this should return an error as no ledger has been set up
```

Comment:
The test name TestRebuildDBsCmd() implies a positive test, but it is actually a negative test to verify an error, should we update the test name to indicate the scenario?

Reply:
The unit test at this level is all about whether the code in this package invokes the actual code in the kvledger package. So, with that intention it's still a positive test. Checking for error case rather helps minimizing the dependency on the kvledger package as it does not need to know internal details of kvledger package. Same for the below comments as well.

Reply:
Ok, given the good code comments, I agree.